.gitignore

```diff
@@ -17,3 +17,6 @@ doc/api/_build
 doc/libraries/*.html
 doc/userguide/RobotFrameworkUserGuide.html
 src/robotframework.egg-info
+log.html
+output.xml
+report.html
```

Comment:
I think we had some reason not to add these files to .gitignore but I don't remember what was it. Probably they could be added, but that should be done as a separate commit/PR outside of this one.